src/platform/forms/tests/forms.unit.spec.js

```diff
@@ -0,0 +1,208 @@
+    reformattedIds.splice(0, 1, VA_FORM_IDS.FORM_10_10EZ);
+    const includedFormIds = configs.map(form => form.formId);
+
+    const allFormIdsSet = new Set(allFormIds);
+    const mappedIdsSet = new Set(mappedIds);
+    expect(allFormIdsSet.size).to.not.lessThan(
+      mappedIdsSet.size,
+      'a schema may have been removed from vets-json-schema/dist/schemas',
+    );
+    expect(allFormIdsSet.size).to.not.greaterThan(
+      mappedIdsSet.size,
+      'a schema may have been added to vets-json-schema/dist/schemas',
+    );
+    expect(allFormIdsSet).to.deep.equal(mappedIdsSet);
+    expect(includedFormIds).to.deep.equal(
+      reformattedIds,
+      'possible missing formId property in a formConfig',
+    );
+  });
+
+  configs.forEach(form => {
+    describe(`${form.formId}:`, () => {
+      describe('migrations:', () => {
+        const { migrations } = form;
+        if (migrations || form.version > 0) {
+          it('should have a length equal to the version number', () => {
+            expect(migrations.length).to.equal(form.version);
+          });
+          it('should be typeof array', () => {
+            expect(migrations).to.be.an('array');
+          });
+          it('should be array of functions', () => {
+            expect(
+              migrations.every(migration => typeof migration === 'function'),
+            ).to.be.true;
+          });
+        }
+      });
+
+      it('should have chapters object', () => {
+        expect(form.chapters).to.be.an('object');
+      });
+
+      it('should have defaultDefinitions object', () => {
+        expect(form.defaultDefinitions).to.be.an('object');
+      });
+
+      it('should have introduction function', () => {
+        expect(form.introduction).to.be.an('function');
+      });
+
+      it('should have prefillEnabled boolean', () => {
+        expect(form.prefillEnabled).to.be.a('boolean');
+      });
+
+      if (form.prefillTransformer) {
+        it('should have prefillTransformer function', () => {
+          expect(form.prefillTransformer).to.be.a('function');
+        });
+      }
+
+      it('should have trackingPrefix', () => {
+        expect(form.trackingPrefix).to.be.a('string');
+      });
+
+      it('should have title', () => {
+        expect(form.title).to.be.a('string');
+      });
+
+      if (form.subTitle) {
+        it('should have subTitle', () => {
+          expect(form.subTitle).to.be.a('string');
+        });
+      }
+
+      it('should have urlPrefix', () => {
+        expect(form.urlPrefix).to.be.a('string');
+      });
+
+      if (form.submitUrl) {
+        it('should have submitUrl', () => {
+          expect(form.submitUrl).to.be.a('string');
+        });
+      }
+
+      if (form.submit) {
+        it('should have submit', () => {
+          expect(form.submit).to.be.a('function');
+        });
+      }
+
+      it('should have savedFormMessages', () => {
+        expect(form.savedFormMessages).to.be.a('object');
```

Comment:
Only if SiP (Save in Progress) is enabled. I'm also not sure if there are default messages. You might be able to remove this test altogether.

Reply:
will wrap test if an if property exists check